Sim_To_DuT_Interface/DuTLogger/DuTLogger.h

```diff
@@ -0,0 +1,135 @@
+/**
+ * DuT-Logger
+ *
+ * Contains the class DuTLogger and all necessary variables.
+ * For better documentation see the class "DuTLogger" and it's functions.
+ *
+ * @author Marco Keul
+ * @version 1.0
+ */
+
+#ifndef TESTLOGGER_DUTLOGGER_H
```

Comment:
I think this guard need to be changed: https://google.github.io/styleguide/cppguide.html#The__define_Guard